src/Generator.Rewrite/Program.cs

```diff
@@ -803,6 +836,154 @@ private static GeneratedVariableIdentifier EmitStringBuilderParameter(MethodDefi
             return generatedVariables;
         }
 
+        private class CountAttribute
+        {
+            public int Count;
+            public string Parameter;
+            public string Computed;
+        }
+
+        private static CountAttribute GetCountAttribute(ParameterDefinition parameter)
+        {
+            var attribute = parameter.CustomAttributes
+                        .FirstOrDefault(a => a.AttributeType.Name == "CountAttribute");
+
+            if (attribute == null)
+            {
+                return new CountAttribute();
+            }
+
+            Func<string, object> getField = delegate(string name) {
+                try
+                {
+                    var field = attribute.Fields.First(f => f.Name == name);
+                    return field.Argument.Value;
+                }
+                catch(Exception ex)
```

Comment:
Swallowing generic exceptions without logging is almost always a bad idea. Can't we catch specific expected exceptions? Or at least log?